src/test/java/entities/PostTest.java

```diff
@@ -30,48 +30,56 @@ public void tearDown() {
     @Test
     public void getFavouritedUsers() {
         LocalDate deadline = LocalDate.of(2018, 12, 31);
+
         Post newPost = new Post(poster, title, mainDesc, tags, collaborators, deadline, 0);
         int user = 0;
         newPost.addFavouritedUser(user);
         assertEquals(user, (int) newPost.getFavouritedUsers().get(0));
+
     }
 
     @Test
     public void getNumFavouritedUsers() {
         LocalDate deadline = LocalDate.of(2018, 12, 31);
+
         Post newPost = new Post(poster, title, mainDesc, tags, collaborators, deadline, 0);
         int user = 0;
         newPost.addFavouritedUser(user);
+
         assertEquals(1, newPost.getNumFavouritedUsers());
     }
 
     @Test
     public void addFavouritedUser() {
         LocalDate deadline = LocalDate.of(2018, 12, 31);
+
         Post newPost = new Post(poster, title, mainDesc, tags, collaborators, deadline, 0);
         int user = 0;
         int user2 = 1;
         newPost.addFavouritedUser(user);
         newPost.addFavouritedUser(user);
         assertEquals(user, (int) newPost.getFavouritedUsers().get(0));
+
         assertEquals(1, newPost.getFavouritedUsers().size());
     }
 
     @Test
     public void removeFavouritedUser() {
         LocalDate deadline = LocalDate.of(2018, 12, 31);
+
         Post newPost = new Post(poster, title, mainDesc, tags, collaborators, deadline, 0);
         int user = 0;
         int user2 = 1;
         newPost.addFavouritedUser(user);
         newPost.removeFavouritedUser(user);
+
         assertEquals(0, newPost.getFavouritedUsers().size());
     }
 
     @Test
     public void setTags() {
         LocalDate deadline = LocalDate.of(2018, 12, 31);
-        Post newPost = new Post(poster, title, mainDesc, tags, collaborators, deadline, 0);
+        Post newPost = new Post(poster.getId(), title, mainDesc, tags, collaborators, deadline, 0);
```

Comment:
Isn't `poster` just an int as declared on line 15? There's no need to call `getId`.

Reply:
I think I resolved this in the merge conflicts, not exactly sure how it works though

Reply:
ah my bad i was looking at an old version where poster was user